jenkins/deploy/Jenkinsfile

```diff
@@ -0,0 +1,80 @@
+pipeline {
+    agent any
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: "5"))
+        timeout time: 60, unit: "MINUTES"
+    }
+
+    stages {
+        stage("Install") {
+            steps {
+                configFileProvider([configFile(fileId: "iis-build-properties", variable: 'BUILD_PROPERTIES')]) {
+                    load "${BUILD_PROPERTIES}"
+                    withEnv(["JAVA_HOME=${ tool type: 'jdk', name: "${JDK_VERSION}" }",
+                             "PATH+MAVEN=${tool type: 'maven', name: "${MAVEN_VERSION}"}/bin:${env.JAVA_HOME}/bin"]) {
+                        sh """
+                            mvn clean install -DskipITs
+                        """
+                    }
+                }
+            }
+        }
+
+        stage("Deploy Primary") {
+            steps {
+                configFileProvider([configFile(fileId: "iis-build-properties", variable: 'BUILD_PROPERTIES')]) {
+                    load "${BUILD_PROPERTIES}"
+                    withEnv(["JAVA_HOME=${ tool type: 'jdk', name: "${JDK_VERSION}" }",
+                             "PATH+MAVEN=${tool type: 'maven', name: "${MAVEN_VERSION}"}/bin:${env.JAVA_HOME}/bin"]) {
+                        sshagent(['iis-deployment-credentials']) {
+                            sh """
+                                cd iis-wf/iis-wf-primary && \
+                                mvn clean package -Poozie-package,deploy \
+                                    -Dworkflow.source.dir=eu/dnetlib/iis/wf/primary/main \
+                                    -DiisConnectionProperties=${WORKSPACE}/connection-properties/iis-cdh5-test-gw.properties
+                            """
+                        }
+                    }
+                }
+            }
+        }
+
+        stage("Release Primary") {
+            steps {
+                script {
+                    def propertiesContent = readFile("${WORKSPACE}/connection-properties/iis-cdh5-test-gw.properties")
+                    def properties = [:]
+                    propertiesContent.split("\\n").each { line ->
+                        if (line.contains('=')) {
+                            def nameAndValue = line.split('=')
+                            properties[nameAndValue[0]] = nameAndValue[1]
+                        }
+                    }
```

Comment:
It was also my first solution, but Jenkins complained that `Properties` cannot be instantiated during job run. I can't find the job output anymore due to log rotation but it was the reason why this manual parsing was needed.

Reply:
Ah, yes, I see. This can be worked around by disabling or reconfiguring the sandbox, but probably better to leave this solution and replace it with `readProperties` when we can.